### PR TITLE
Adds the atmos type: NoDecay

### PR DESCRIPTION
**Pull request by EmeraldCandy**
## What Does This PR Do
Adds a new MILLA type to prepare for a new station map. This zone does not have temperature decay

## Why It's Good For The Game
Prepares for a future map addition, and a no-temp-decay type could be useful at later dates.

## Testing
Made a special floor type with the new MILLA mode. Put different gasses into the room, measuring their temp decay. No decay noted. Applies gasses to normal terfs and observed temperature decay

<hr>

## Changelog
:cl:
add: Added NoDecay to MILLA
/:cl:

### CODE REVIEW

**Comment by EmeraldCandy** [comment]
!build_milla

**Review by AffectedArc07 — DISMISSED** [review]
Could you also stick `#define ATMOS_MODE_NODECAY 3` into `code/__DEFINES/atmospherics_defines.dm`?

**Comment by EmeraldCandy** [comment]
> Could you also stick `#define ATMOS_MODE_NODECAY 3` into `code/__DEFINES/atmospherics_defines.dm`?

yup just did an oopsie and unticked it from the commit list

**Comment by EmeraldCandy** [comment]
added testing

**Review by AffectedArc07 — APPROVED** [review]
Approved what I have

**Review by FunnyMan3595 — APPROVED** [review]

File: code/__DEFINES/atmospherics_defines.dm [hunk 1]
```diff
@@ -156,6 +156,7 @@
 #define ATMOS_MODE_SPACE 0
 #define ATMOS_MODE_SEALED 1
 #define ATMOS_MODE_EXPOSED_TO_ENVIRONMENT 2
+#define ATMOS_MODE_NO_DECAY 3
```
Comment (lewcc) [inline code/__DEFINES/atmospherics_defines.dm:159]: Could you add a comment here to indicate what this means on the DM side?

Comment (EmeraldCandy) [reply code/__DEFINES/atmospherics_defines.dm]: done!

File: code/__DEFINES/atmospherics_defines.dm [hunk 2]
```diff
@@ -156,6 +156,7 @@
 #define ATMOS_MODE_SPACE 0
 #define ATMOS_MODE_SEALED 1
 #define ATMOS_MODE_EXPOSED_TO_ENVIRONMENT 2
+#define ATMOS_MODE_NO_DECAY 3 //Does not try to normalize temperature to the environment type temp
```
Comment (FunnyMan3595) [inline code/__DEFINES/atmospherics_defines.dm:159]:
```suggestion
#define ATMOS_MODE_NO_DECAY 3 // Prevents hot tiles from automatically decaying towards T20C.
```

**Comment by EmeraldCandy** [comment]
!build_milla

**Review by DGamerL — APPROVED** [review]

**Comment by DGamerL** [comment]
PR has been deconflicted

**Pull request merged**